packages/api-kit/src/utils/signDelegate.ts

```diff
@@ -1,10 +1,14 @@
-import { Signer } from 'ethers'
+import { Chain, LocalAccount, Transport, WalletClient } from 'viem'
```

Comment:
Should we use Account instead LocalAccount in all the places?

An Account can be both LocalAccount | JsonRPCAccount

Reply:
Yes definitely!